ref-exchange/src/simple_pool.rs

```diff
@@ -49,19 +50,19 @@ impl SimplePool {
             total_fee < FEE_DIVISOR && (exchange_fee + referral_fee) <= total_fee,
             "ERR_FEE_TOO_LARGE"
         );
-        assert_ne!(token_account_ids.len(), 1, "ERR_NOT_ENOUGH_TOKENS");
-        assert!(
-            token_account_ids.len() <= MAX_NUM_TOKENS,
-            "ERR_TOO_MANY_TOKENS"
-        );
+        // [AUDIT_10]
+        assert_eq!(token_account_ids.len(), MAX_NUM_TOKENS, "ERR_SHOULD_HAVE_2_TOKENS");
```

Comment:
let's rename this to NUM_TOKENS? as there is only 2 allowed either way.

Reply:
OK